### DISCUSSION TITLE
Organization naming

### DISCUSSION THREAD

**Discussion by worthant** [Imtjl/.github#1] [Ideas]
So, i am struggling so much with our organization name. What should it be?) 
Assemble? SCN(Students collaborative network)? Classic? I need your original ideas!)

**Comment by khabner** [comment]
So can we remove the names of real organizations from Markdowns? Also I think it isn't really necessary to change or not the name of organization.

**Reply by worthant** [reply]
I think that i can change our organization name in every markdown, if needed.
It isn't so hard, because we have little to any markdowns)
